elfi/methods/parameter_inference.py

```diff
@@ -936,54 +935,87 @@ def _report_batch(self, batch_index, params, distances):
             str += "{}{} at {}\n".format(fill, distances[i].item(), params[i])
         logger.debug(str)
 
-    def plot_state(self, **options):
-        """Plot the GP surface
-        
-        This feature is still experimental and currently supports only 2D cases.
+    def plot_acq_points(self):
```

Comment:
Could the majority of this code be moved to the visualization files? It would be ideal if this had a line or two of code in this file. This function could also be useful in the GPyModel class.